firewood/tests/v2api.rs

```diff
@@ -1,15 +1,15 @@
 // Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE.md for licensing terms.
 
-use std::{error::Error, path::PathBuf, sync::Arc};
```

Comment:
Why?

Reply:
We need the other `v2::api::Error` type and this one conflicts. I'll clean this up some and make it clearer.